lib/Translation/Verilog/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 file(GLOB globbed *.cpp)
 
-add_circt_library(CIRCTEmitVerilog
```

Comment:
We don't have it today, but conceptually we could have a verilog parser.  That is a very different thing than a verilog printer though, in terms of clients and dependencies, so these should end up living in different libraries.  

As such, I think that `CIRCTTranslationVerilog` is too generic of a name for this.

Reply:
Good point, for some reason I was thinking we would build import and export into the same library. How about `CIRCTExportVerilog`, and the parser would be `CIRCTImportVerilog`?

Reply:
Works for me!